views/rsvp.hbs

```diff
@@ -1 +1,52 @@
-<h2>RSVP</h2>
+<div ng-controller="weddingTime.guests.RsvpController">
+    <div class="jumbotron" ng-hide="guest.invitationName" ng-show="attempts">
+        <h1>Hello! Got an RSVP code?</h1>
+        <p>Check your invitation for your unique RSVP code and enter it below.</p>
+        <div class="row">
+            <div class="col-lg-6">
+                <div class="input-group">
+                    <input type="text" class="form-control" placeholder="ex: farmhouse" ng-model="rsvpCode">
+                    <span class="input-group-btn">
+                        <button class="btn btn-primary" type="button" ng-click="validate(rsvpCode)">Go!</button>
+                    </span>
+                </div>
+            </div>
+        </div>
+        <a href="mailto:mbarlock@gmail.com,edurham215@gmail.com?Subject=Wedding%20RSVP" ng-show="attempts" ng-cloak>Having Trouble? Shoot us an email.</a>
```

Comment:
Make this configurable through an env variable please, I'd rather not put my/liz's email on github. After thinking about how to do this its not super obvious how to do this though... We can bounce ideas off each other if you can't think of anything